windows-installer/auto-update.ps1

```diff
@@ -0,0 +1,134 @@
+[CmdletBinding()]
+param()
+begin {}
+process {
+    New-EventLog -Source "certbot/auto-update.ps1" -LogName "CertbotAutoUpdate" -ErrorAction SilentlyContinue
+
+    function Write-Message($message, $level = "Information") {
+        Write-EventLog -Source "certbot/auto-update.ps1" -LogName "CertbotAutoUpdate" -EventID 1 -EntryType $level -Message $message
+        Write-Host $message
+    }
+
+    function Write-Error($message) {
```

Comment:
nit: Can we change the name of this function slightly to reflect that this also raises an exception? Maybe something like `Error`, `Fatal`, or `Exit`?

Reply:
Done.